package.json

```diff
@@ -1,28 +1,34 @@
 {
-  "name": "subql-nodle",
-  "version": "0.0.4",
-  "description": "This project can be use as a starting po...",
+  "name": "subql-dictionary",
```

Comment:
I thought our dictionary is the one located here: https://github.com/subquery/subql-dictionary/tree/main/nodle-parachain. Why we should change the name of the project to dictionary here?